### PR TITLE
Fix latex build for #828

### PR DESCRIPTION
**Pull request by PeterC-DLS**
Correct indentation in NXmx and raise maxdepth list as some application definitions are very deeply nested

### CODE REVIEW

File: manual/source/conf.py [hunk 1]
```diff
@@ -82,3 +82,8 @@
 
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'NeXusManualdoc'
+
```
Comment (prjemian) [inline manual/source/conf.py:85]: Aha!  testing it now...

**Comment by prjemian** [comment]
The build (`make local all`) finishes with these lines on the console:

```
Output written on nexus.pdf (400 pages, 5561864 bytes).
Transcript written on nexus.log.
Latexmk: Index file 'nexus.idx' was written
Latexmk: References changed.
Latexmk: Log file says output to 'nexus.pdf'
Latexmk: Errors, so I did not complete making targets
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 256
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
Makefile:29: recipe for target 'nexus.pdf' failed
make[2]: *** [nexus.pdf] Error 12
make[2]: Leaving directory '/home/prjemian/Documents/projects/NeXus/definitions/build/manual/build/latex'
Makefile:26: recipe for target 'latexpdf' failed
make[1]: *** [latexpdf] Error 2
make[1]: Leaving directory '/home/prjemian/Documents/projects/NeXus/definitions/build/manual'
# finally, report what was built
HTML built: -rw-rw-r-- 1 prjemian 12K Oct 20 13:20 build/manual/build/html/index.html
PDF built: -rw-rw-r-- 1 prjemian 5.4M Oct 20 13:21 build/manual/build/latex/nexus.pdf
```

But there is no report about nested too deeply:

    grep nested build/manual/build/latex/nexus.log

and the PDF looks OK.

LGTM

**Review by prjemian — APPROVED** [review]
as commented

**Comment by PeterC-DLS** [comment]
Errors are all similar to:
```
] (./nexus.ind [382]
! Undefined control sequence.
<argument> \spxpagem

l.35            \hyperindexformat{\spxpagem}{158}

The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.
```

**Comment by prjemian** [comment]
@PeterC-DLS Merge when you are ready.

**Comment by PeterC-DLS** [comment]
Above [comment](https://github.com/nexusformat/definitions/pull/844#issuecomment-713147211) does not cause #828, so merging this PR.

**Pull request merged**